src/errors.rs

```diff
@@ -193,6 +194,11 @@ error_chain! {
             description("invalid custom toolchain name")
             display("invalid custom toolchain name: '{}'", t)
         }
+        InvalidProfile(t: String) {
+            description("invalid profile name")
+            display("invalid profile name: '{}'; valid names are: \
+                     `minimal`, `default`, and `complete`", t)
```

Comment:
Given below you show how to keep the names in sync, use the same logic to manually list valid profile names here?